templates/zerver/api/update-message-flags.md

```diff
@@ -58,6 +58,58 @@ curl -X POST {{ api_url }}/v1/messages/flags \
 
 {generate_api_arguments_table|zulip.yaml|/messages/flags:post}
 
+## Available Flags
+<div>
+    <table>
```

Comment:
FYI markdown have its own table syntax.

Reply:
It does, though the markdown table format is pretty painful for something with paragraphs in it like this.